test/integration/testdata/egress-proxy.yaml.tmpl

```diff
@@ -41,10 +41,10 @@ spec:
         - -v
         - "{{.Verbosity}}"
         - --discoveryAddress
-        - istio-pilot.{{.IstioNamespace}}:8080
+        - istio-pilot:8080
 {{if .Zipkin}}
         - --zipkinAddress
-        - zipkin.{{.IstioNamespace}}:9411
+        - zipkin:9411
```

Comment:
General remark about zipkin, unrelated to the changes in this PR:

If the user has zipkin already deployed in another namespace, he will not be able to configure Istio to use it, even if he changes the configMap, because this is hardcoded in the templates.

Reply:
ConfigMap is also a template. We need folks to change zipkin address in two places in the installation yaml.